lib/fray/src/fray/cluster/ray/dashboard.py

```diff
@@ -408,10 +490,25 @@ def ray_dashboard(config: DashboardConfig) -> Generator[DashboardConnection, Non
 
         # Set new values
         os.environ["RAY_ADDRESS"] = f"http://localhost:{ports.dashboard_port}"
-        os.environ["RAY_API_SERVER_ADDRESS"] = f"ray://localhost:{ports.api_port}"
+        # Ray's Jobs SDK treats RAY_API_SERVER_ADDRESS as an override for the
+        # dashboard URL and will resolve Ray Client (ray://) addresses to the
+        # head node's internal webui_url (e.g. http://10.x.x.x:8265), which is
+        # not reachable from a developer laptop. Keep this as an HTTP URL so
+        # job submission uses the SSH-forwarded dashboard port.
+        os.environ["RAY_API_SERVER_ADDRESS"] = f"http://localhost:{ports.dashboard_port}"
```

Comment:
So did this behavior change with the new Ray version? Why did it work before I wonder?